src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -213,28 +233,34 @@ public Visitable visit(ClassOrInterfaceDeclaration node, Void arg) {
 
   @Override
   public Visitable visit(ExplicitConstructorInvocationStmt node, Void arg) {
-    NodeList<Expression> arguments = node.getArguments();
-    List<String> parametersList = new ArrayList<>();
-    for (Expression parameter : arguments) {
-      if (!canBeSolved(parameter)) {
-        return super.visit(node, arg);
-      }
-      ResolvedType type = parameter.calculateResolvedType();
-      if (type instanceof PrimitiveType) {
-        parametersList.add(type.asPrimitive().name());
-      } else if (type instanceof ReferenceType) {
-        parametersList.add(type.asReferenceType().getQualifiedName());
+    try {
```

Comment:
This is supposed to be in another PR. This part is to fix a bug with super call. But I think it is quite simple so I put it in this PR too. This codes make sure that Specimin will only create synthetic files for the parent class if the parent class is not solvable, which is quite easy to understand. As we already have the `SuperClass` test case where the parent class is solvable, I don't add any more test cases for this bug.

Reply:
Please add this information to the code as a comment: it's not obvious from the code that the purpose of the call to `node.resolve().getQualifiedSignature()` exists to check if the parent is solvable.